server/middleware/classrooms.coffee

```diff
@@ -151,7 +162,7 @@ module.exports =
     code = req.body.code.toLowerCase()
     classroom = yield Classroom.findOne({code: code})
     if not classroom
-      throw new errors.NotFound(res) 
+      throw new errors.NotFound(res)
```

Comment:
👍 

BTW, you wrote that line 6 days ago: https://github.com/codecombat/codecombat/blame/master/server/middleware/classrooms.coffee#L154